templates/core/resource_list.html

```diff
@@ -51,6 +51,21 @@ <h5 class="card-title w-100">{{ resource.title }} <i class="bi bi-folder-symlink
                   </a>
                   {% endfor %}
                 </div>
+
+              {% for user in users %}
```

Comment:
As I have already explained that you do not need to send all `users` in template as `context`, you can simply access the resource creator users by this template code -
```
{{ resource.added_by.full_name }}
```